src/common/peer.cc

```diff
@@ -68,22 +68,34 @@ namespace Pistache::Tcp
     {
         if (hostname_.empty())
         {
-            char host[NI_MAXHOST];
-            struct sockaddr_in sa;
-            sa.sin_family = AF_INET;
-            if (inet_pton(AF_INET, addr.host().c_str(), &sa.sin_addr) == 0)
+            if (addr.family() == AF_UNIX)
             {
-                hostname_ = addr.host();
+                //
+                // Communication through unix domain sockets is constrained to
+                // the local host.
+                //
+                hostname_.assign("localhost");
```

Comment:
I have slight concerns about this.  Maybe return an empty string, or a value that is NOT a valid hostname, like "**unix" or similar.

Reply:
"[Unix]" ?
We already use "[Abstract]" for linux abstract sockets.
However, "localhost" matches the expectations as in - all communication for unix domain sockets happens within localhost.
Any idea how that could be used in the way other than for presentation purposes?